front/lib/api/assistant/global_agents.ts

```diff
@@ -32,6 +36,13 @@ import { prodAPICredentialsForOwner } from "@app/lib/auth";
 import { GlobalAgentSettings } from "@app/lib/models/assistant/agent";
 import logger from "@app/logger/logger";
 
+// Used when returning an agent with status 'disabled_by_admin'
+const dummyModelConfiguration = {
+  providerId: GPT_3_5_TURBO_MODEL_CONFIG.providerId,
+  modelId: GPT_3_5_TURBO_MODEL_CONFIG.modelId,
+  temperature: 0,
+};
+
```

Comment:
@spolu seems a bit hackish but `AgentConfigurationType` requires an `AgentModelConfigurationType`. Should we leave it like this or should I alter `AgentConfigurationType` to allow for `null` if scope is `disabled_by_admin`?

Reply:
I think this is fiiiiine